File: Pidev_EGOV/Pi_Egov_Java/src/esprit/egov/entities/CarteEdinar.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package esprit.egov.entities;

import java.sql.Date;


/**
 *
 * @author USER
 */
public class CarteEdinar {
    private int num_carte;
    private int code_carte;
    private int solde_actuel;
    private Date date_dernier_retirement;
    private int dernier_retirement;
    private Date date_creation;
    private Date date_dernier_versement;
    private int dernier_versement;
    private int validator_creation;
    private int validator_recuperation;
    private int validator_changement;
    private int validator_renouvellement;
    private int validator_blockage;
    private int validator_deblockage;
    private String reponseq1;
    private String reponseq2;
    private String reponseq3;

    public CarteEdinar(int num_carte, int code_carte, int solde_actuel, Date date_dernier_retirement, int dernier_retirement, Date date_creation, Date date_dernier_versement, int dernier_versement, int validator_creation, int validator_recuperation, int validator_changement, int validator_renouvellement, int validator_blockage, int validator_deblockage, String reponseq1, String reponseq2, String reponseq3) {
        this.num_carte = num_carte;
        this.code_carte = code_carte;
        this.solde_actuel = solde_actuel;
        this.date_dernier_retirement = date_dernier_retirement;
        this.dernier_retirement = dernier_retirement;
        this.date_creation = date_creation;
        this.date_dernier_versement = date_dernier_versement;
        this.dernier_versement = dernier_versement;
        this.validator_creation = validator_creation;
        this.validator_recuperation = validator_recuperation;
        this.validator_changement = validator_changement;
        this.validator_renouvellement = validator_renouvellement;
        this.validator_blockage = validator_blockage;
        this.validator_deblockage = validator_deblockage;
        this.reponseq1 = reponseq1;
        this.reponseq2 = reponseq2;
        this.reponseq3 = reponseq3;
    }

    public CarteEdinar(int num_carte, int code_carte, int solde_actuel, Date date_creation, Date date_dernier_versement, int dernier_versement) {
        this.num_carte = num_carte;
        this.code_carte = code_carte;
        this.solde_actuel = solde_actuel;
        this.date_creation = date_creation;
        this.date_dernier_versement = date_dernier_versement;
        this.dernier_versement = dernier_versement;
    }

    public CarteEdinar() {
    }

    public int getNum_carte() {
        return num_carte;
    }

    public void setNum_carte(int num_carte) {
        this.num_carte = num_carte;
    }

    public int getCode_carte() {
        return code_carte;
    }

    public void setCode_carte(int code_carte) {
        this.code_carte = code_carte;
    }

    public int getSolde_actuel() {
        return solde_actuel;
    }

    public void setSolde_actuel(int solde_actuel) {
        this.solde_actuel = solde_actuel;
    }

    public Date getDate_dernier_retirement() {
        return date_dernier_retirement;
    }

    public void setDate_dernier_retirement(Date date_dernier_retirement) {
        this.date_dernier_retirement = date_dernier_retirement;
    }

    public int getDernier_retirement() {
        return dernier_retirement;
    }

    public void setDernier_retirement(int dernier_retirement) {
        this.dernier_retirement = dernier_retirement;
    }

    public Date getDate_creation() {
        return date_creation;
    }

    public void setDate_creation(Date date_creation) {
        this.date_creation = date_creation;
    }

    public Date getDate_dernier_versement() {
        return date_dernier_versement;
    }

    public void setDate_dernier_versement(Date date_dernier_versement) {
        this.date_dernier_versement = date_dernier_versement;
    }

    public int getDernier_versement() {
        return dernier_versement;
    }

    public void setDernier_versement(int dernier_versement) {
        this.dernier_versement = dernier_versement;
    }

    public int getValidator_creation() {
        return validator_creation;
    }

    public void setValidator_creation(int validator_creation) {
        this.validator_creation = validator_creation;
    }

    public int getValidator_recuperation() {
        return validator_recuperation;
    }

    public void setValidator_recuperation(int validator_recuperation) {
        this.validator_recuperation = validator_recuperation;
    }

    public int getValidator_changement() {
        return validator_changement;
    }

    public void setValidator_changement(int validator_changement) {
        this.validator_changement = validator_changement;
    }

    public int getValidator_renouvellement() {
        return validator_renouvellement;
    }

    public void setValidator_renouvellement(int validator_renouvellement) {
        this.validator_renouvellement = validator_renouvellement;
    }

    public int getValidator_blockage() {
        return validator_blockage;
    }

    public void setValidator_blockage(int validator_blockage) {
        this.validator_blockage = validator_blockage;
    }

    public int getValidator_deblockage() {
        return validator_deblockage;
    }

    public void setValidator_deblockage(int validator_deblockage) {
        this.validator_deblockage = validator_deblockage;
    }

    public String getReponseq1() {
        return reponseq1;
    }

    public void setReponseq1(String reponseq1) {
        this.reponseq1 = reponseq1;
    }

    public String getReponseq2() {
        return reponseq2;
    }

    public void setReponseq2(String reponseq2) {
        this.reponseq2 = reponseq2;
    }

    public String getReponseq3() {
        return reponseq3;
    }

    public void setReponseq3(String reponseq3) {
        this.reponseq3 = reponseq3;
    }

    @Override
    public int hashCode() {
        int hash = 3;
        hash = 71 * hash + this.num_carte;
        hash = 71 * hash + this.code_carte;
        return hash;
    }

    @Override
    public boolean equals(Object obj) {
        if (obj == null) {
            return false;
        }
        if (getClass() != obj.getClass()) {
            return false;
        }
        final CarteEdinar other = (CarteEdinar) obj;
        if (this.num_carte != other.num_carte) {
            return false;
        }
        if (this.code_carte != other.code_carte) {
            return false;
        }
        return true;
    }
    
    
}

File: Pidev_EGOV/EGOV_symfony/src/MyApp/UserBundle/Entity/CarteEdinar.php
<?php

namespace MyApp\UserBundle\Entity;

use Doctrine\ORM\Mapping as ORM;
use Symfony\Component\Validator\Constraint as Assert;

/**
 * CarteEdinar
 *
 * @ORM\Table(name="carte_edinar")
 * @ORM\Entity
 */
class CarteEdinar {

    /**
     * Assert\NotBlank()
     * @var integer
     *
     * @ORM\Column(name="num_carte", type="integer", nullable=false)
     * @ORM\Id
     * @ORM\GeneratedValue(strategy="IDENTITY")
     */
    private $numCarte;

    /**
     * Assert\NotBlank()
     * @var integer
     * @ORM\Column(name="code_carte", type="integer", nullable=false)
     */
    private $codeCarte;

    /**
     * @var integer
     *
     * @ORM\Column(name="solde_actuel", type="integer", nullable=true)
     */
    private $soldeActuel;

    /**
     * @var \DateTime
     *
     * @ORM\Column(name="date_dernier_retirement", type="date", nullable=true)
     */
    private $dateDernierRetirement;

    /**
     * @var integer
     *
     * @ORM\Column(name="dernier_retirement", type="integer", nullable=true)
     */
    private $dernierRetirement;

    /**
     * @var \DateTime
     *
     * @ORM\Column(name="date_creation", type="date", nullable=true)
     */
    private $dateCreation;

    /**
     * @var \DateTime
     *
     * @ORM\Column(name="date_dernier_versement", type="date", nullable=true)
     */
    private $dateDernierVersement;

    /**
     * @var integer
     *
     * @ORM\Column(name="dernier_versement", type="integer", nullable=true)
     */
    private $dernierVersement;

    /**
     * @var integer
     *
     * @ORM\Column(name="validator_creation", type="integer", nullable=true)
     */
    private $validator_creation;

    /**
     * @var integer
     *
     * @ORM\Column(name="validator_recuperation", type="integer", nullable=true)
     */
    private $validator_recuperation;

    /**
     * @var integer
     *
     * @ORM\Column(name="validator_changement", type="integer", nullable=true)
     */
    private $validator_changement;

    /**
     * @var integer
     *
     * @ORM\Column(name="validator_renouvellement", type="integer", nullable=true)
     */
    private $validator_renouvellement;

    /**
     * @var integer
     *
     * @ORM\Column(name="validator_blockage", type="integer", nullable=true)
     */
    private $validator_blockage;

    /**
     * @var integer
     *
     * @ORM\Column(name="validator_deblockage", type="integer", nullable=true)
     */
    private $validator_deblockage;

    /**
     * @var string
     *
     * @ORM\Column(name="reponseq1", type="string", length=30, nullable=true)
     */
    private $reponseq1;

    /**
     * @var string
     *
     * @ORM\Column(name="reponseq2", type="string", length=30, nullable=true)
     */
    private $reponseq2;

    /**
     * @var string
     *
     * @ORM\Column(name="reponseq3", type="string", length=30, nullable=true)
     */
    private $reponseq3;

    /**
     * Get numCarte
     *
     * @return string 
     */
    public function getNumCarte() {
        return $this->numCarte;
    }

    /**
     * Set codeCarte
     *
     * @param integer $codeCarte
     * @return CarteEdinar
     */
    public function setCodeCarte($codeCarte) {
        $this->codeCarte = $codeCarte;

        return $this;
    }

    /**
     * Get codeCarte
     *
     * @return integer 
     */
    public function getCodeCarte() {
        return $this->codeCarte;
    }

    /**
     * Set soldeActuel
     *
     * @param integer $soldeActuel
     * @return CarteEdinar
     */
    public function setSoldeActuel($soldeActuel) {
        $this->soldeActuel = $soldeActuel;

        return $this;
    }

    /**
     * Get soldeActuel
     *
     * @return integer 
     */
    public function getSoldeActuel() {
        return $this->soldeActuel;
    }

    /**
     * Set dateDernierRetirement
     *
     * @param \DateTime $dateDernierRetirement
     * @return CarteEdinar
     */
    public function setDateDernierRetirement($dateDernierRetirement) {
        $this->dateDernierRetirement = $dateDernierRetirement;

        return $this;
    }

    /**
     * Get dateDernierRetirement
     *
     * @return \DateTime 
     */
    public function getDateDernierRetirement() {
        return $this->dateDernierRetirement;
    }

    /**
     * Set dernierRetirement
     *
     * @param integer $dernierRetirement
     * @return CarteEdinar
     */
    public function setDernierRetirement($dernierRetirement) {
        $this->dernierRetirement = $dernierRetirement;

        return $this;
    }

    /**
     * Get dernierRetirement
     *
     * @return integer 
     */
    public function getDernierRetirement() {
        return $this->dernierRetirement;
    }

    /**
     * Set dateCreation
     *
     * @param \DateTime $dateCreation
     * @return CarteEdinar
     */
    public function setDateCreation($dateCreation) {
        $this->dateCreation = $dateCreation;

        return $this;
    }

    /**
     * Get dateCreation
     *
     * @return \DateTime 
     */
    public function getDateCreation() {
        return $this->dateCreation;
    }

    /**
     * Set dateDernierVersement
     *
     * @param \DateTime $dateDernierVersement
     * @return CarteEdinar
     */
    public function setDateDernierVersement($dateDernierVersement) {
        $this->dateDernierVersement = $dateDernierVersement;

        return $this;
    }

    /**
     * Get dateDernierVersement
     *
     * @return \DateTime 
     */
    public function getDateDernierVersement() {
        return $this->dateDernierVersement;
    }

    /**
     * Set dernierVersement
     *
     * @param integer $dernierVersement
     * @return CarteEdinar
     */
    public function setDernierVersement($dernierVersement) {
        $this->dernierVersement = $dernierVersement;

        return $this;
    }

    /**
     * Get dernierVersement
     *
     * @return integer 
     */
    public function getDernierVersement() {
        return $this->dernierVersement;
    }

    /**
     * Set validator_creation
     *
     * @param integer $validator_creation
     * @return CarteEdinar
     */
    public function setValidatorCreation($validator_creation) {
        $this->validator_creation = $validator_creation;

        return $this;
    }

    /**
     * Get validator_creation
     *
     * @return integer 
     */
    public function getValidatorCreation() {
        return $this->validator_creation;
    }

    /**
     * Set validator_recuperation
     *
     * @param integer $validator_recuperation
     * @return CarteEdinar
     */
    public function setValidatorRecuperation($validator_recuperation) {
        $this->validator_recuperation = $validator_recuperation;

        return $this;
    }

    /**
     * Get validator_recuperation
     *
     * @return integer 
     */
    public function getValidatorRecuperation() {
        return $this->validator_recuperation;
    }

    /**
     * Set validator_changement
     *
     * @param integer $validator_changement
     * @return CarteEdinar
     */
    public function setValidatorChangement($validator_changement) {
        $this->validator_changement = $validator_changement;

        return $this;
    }

    /**
     * Get validator_changement
     *
     * @return integer 
     */
    public function getValidatorChangement() {
        return $this->validator_changement;
    }

    /**
     * Set validator_renouvellement
     *
     * @param integer $validator_renouvellement
     * @return CarteEdinar
     */
    public function setValidatorRenouvellement($validator_renouvellement) {
        $this->validator_renouvellement = $validator_renouvellement;

        return $this;
    }

    /**
     * Get validator_renouvellement
     *
     * @return integer 
     */
    public function getValidatorRenouvellement() {
        return $this->validator_renouvellement;
    }

    /**
     * Set validator_blockage
     *
     * @param integer $validator_blockage
     * @return CarteEdinar
     */
    public function setValidatorBlockage($validator_blockage) {
        $this->validator_blockage = $validator_blockage;

        return $this;
    }

    /**
     * Get validator_blockage
     *
     * @return integer 
     */
    public function getValidatorBlockage() {
        return $this->validator_blockage;
    }

    /**
     * Set validator_deblockage
     *
     * @param integer $validator_deblockage
     * @return CarteEdinar
     */
    public function setValidatorDeblockage($validator_deblockage) {
        $this->validator_deblockage = $validator_deblockage;

        return $this;
    }

    /**
     * Get validator_deblockage
     *
     * @return integer 
     */
    public function getValidatorDeblockage() {
        return $this->validator_deblockage;
    }

    /**
     * Set reponseq1
     *
     * @param string $reponseq1
     * @return CarteEdinar
     */
    public function setReponseq1($reponseq1) {
        $this->reponseq1 = $reponseq1;

        return $this;
    }

    /**
     * Get reponseq1
     *
     * @return string 
     */
    public function getReponseq1() {
        return $this->reponseq1;
    }
     /**
     * Set reponseq2
     *
     * @param string $reponseq2
     * @return CarteEdinar
     */
    public function setReponseq2($reponseq2) {
        $this->reponseq2 = $reponseq2;

        return $this;
    }

    /**
     * Get reponseq2
     *
     * @return string 
     */
    public function getReponseq2() {
        return $this->reponseq2;
    }
        /**
     * Set reponseq3
     *
     * @param string $reponseq3
     * @return CarteEdinar
     */
    public function setReponseq3($reponseq3) {
        $this->reponseq3 = $reponseq3;

        return $this;
    }

    /**
     * Get reponseq3
     *
     * @return string 
     */
    public function getReponseq3() {
        return $this->reponseq3;
    }
}
